topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml

```diff
@@ -0,0 +1,596 @@
+      in the alignment that only contain gap characters (i.e. "."). These can be
+      pulled out without losing any information. We'll do all this with
+      "Filter.seqs".<br><br> Search and open the "Filter.seqs" tool'
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_filter_seqs%2Fmothur_filter_seqs%2F1.36.1.0"]
+  - title: Filter sequences
+    element: '#center-panel'
+    content: |-
+      Execute the tool with <ul>
+        <li>"fasta" to "good.fasta" output from "Screen.seqs"</li>
+        <li>"vertical" to "Yes"</li>
+        <li>"trump" to "."</li>
+      </ul>
+    placement: left
+  - title: Extraction of taxonomic information
+    content: >-
+      The main questions when analyzing amplicon data are: Which micro-organisms
+      are present in an environmental samples? And in which proportion? What is
+      the structure of the community of the micro-organisms?<br><br> The idea is
+      to take the sequences and assign them to a taxon. To do that, we group (or
+      cluster) sequences based on their similarity to define Operational
+      Taxonomic Units (OTUs); groups of similar sequences that can be treated as
+      a single “genus” or “species” (depending on the clustering threshold)
+    backdrop: true
+  - title: Perform preliminary clustering of sequences and remove undesired sequences
+    element: '#tool-search-query'
+    content: >-
+      The first thing we want to do is to further de-noise our sequences, by
+      pre-clustering the sequences using the "Pre.cluster" command, allowing for
+      up to 2 differences between sequences. This command will split the
+      sequences by group and then sort them by abundance and go from most
+      abundant to least and identify sequences that differ no more than 2
+      nucleotides from on another. If this is the case, then they get merged. We
+      generally recommend allowing 1 difference for every 100 basepairs of
+      sequence.<br><br> Search and open the "Pre.cluster" tool
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_pre_cluster%2Fmothur_pre_cluster%2F1.36.1.0"]   
+  - title: Perform preliminary clustering of sequences and remove undesired sequences
+    element: '#center-panel'
+    content: |-
+      Execute the tool with <ul>
+        <li>"fasta" to the fasta output from the last "Filter.seqs" run</li>
+        <li>"name file or count table" to the count table from the last "Screen.seqs" step</li>
+        <li>"diffs" to 2</li>
+      </ul>
+    placement: left
+  - title: Perform preliminary clustering of sequences and remove undesired sequences
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      How many unique sequences are we left with after this clustering of highly
+      similar sequences?
+    placement: left
+  - title: Sequence Alignment
```

Comment:
think this doesn't belong here? already present at L233 before the alignment step?